test/quicer_SUITE.erl

```diff
@@ -537,7 +537,10 @@ simple_stream_server(Owner, Config, Port) ->
   end.
 
 default_conn_opts() ->
-  [{alpn, ["sample"]}].
+  [{alpn, ["sample"]},
+   {idle_timeout_ms, 5000},
+   {peer_unidi_stream_count, 1},
+   {peer_bidi_stream_count, 10}].
```

Comment:
FYI,  We need 'set socket opt' API in place to test this parm.

Test scenario:

1. start connection with peer_bidi_stream_count 1, 

2. start stream one and send data over it.
    The receiver recv the data

3. start stream two and send data over it.
    the receiver does not recv the data

4. set socket opt to peer_bidi_stream_count 10

5. receiver recv data from stream 2.

Reply:
maybe my next task?